src/olympia/abuse/urls.py

```diff
@@ -1,9 +1,8 @@
-from django.conf.urls import include, url
+from views import AddonAbuseViewSet, UserAbuseViewSet
```

Comment:
having relative olympia imports before imported packages and django seems wrong to me.